### ISSUE TITLE
Deploy Your First SAPUI5 App

### ISSUE THREAD

**Issue by Lorenz-M** [sap-tutorials/Tutorials#18036]
Tutorials: https://developers.sap.com/tutorials/appstudio-sapui5-create.html
--------------------------
Greetings,
The Tutorial was great. Just two steps were missing:
"In there, select the generated folder sapui5 and hit open to see the new project."
-> the format should be changed to "all files" first

"Click on the bottom-left corner of the screen to start the connection flow."
-> first you have to go into Preferences -> Extensions -> Cloud Foundry Tools -> and check Show Target Information

Thanks again for the Tutorial!
Best Regards
Lorenz M

**Comment by cjbernal** [comment]
Hi Lorenz,

We're glad you enjoyed the tutorial! Thank you for taking the time to share your thoughts and help improve the tutorial for others. I took a walk through it to better understand your feedback and here's what I found:

On `All files`: Selecting `All files` didn't seem necessary to me -- the project was opened in my workspace without it. But your way works too :-)

On connection flow: I see how this can be confusing to learners. It appears the flow and default settings have changed since this tutorial was written. I will work with the team to see how we can improve this step since the default flow has changed. 

Kind regards,
Conrad (SAP UX Engineering)
